package.json

```diff
@@ -10,10 +10,11 @@
         "@types/mocha": "^9.0.0",
         "chai": "^4.3.4",
         "mocha": "^9.0.3",
+        "pg": "^8.11.0",
         "prettier": "^2.6.2",
         "redis": "^4.6.7",
         "redis-om": "^0.3.6",
```

Comment:
Maybe drop `redis` and `redis-om` from the deps if we aren't including the indexer anymore?